smart-on-fhir/0-introduction/introduction.md

```diff
@@ -1,45 +1,34 @@
+Estimated time: 1 hour and 35 minutes
 
-*You may add an option video, using this format: [](youtube:YouTube video id)*
+### During this workshop, we will cover the following steps:
 
-  [](youtube:zNKxJjkq0Pw)
-
-### Objectives
-
-*List objectives for the workshop*
-
-In this workshop, you will learn how to:
-* Provision
-* Setup
-* Load
-* Query
+* Introduction: We will provide an overview of the workshop's objectives, discuss the importance of ASCVD risk assessment, and explain the tools and technologies we'll be using.
+* Signing up for an APEX Workspace: Before we dive into application development, we'll walk you through the process of signing up for an APEX workspace, which will serve as your development environment for building the ASCVD Risk Calculator.
+* APEX Application Setup: Once you have your APEX workspace ready, we'll guide you in setting up a new application. We'll explore the application builder, create the necessary database objects, and establish the foundation for our ASCVD Risk Calculator.
+* Signing up for Code Console: The Code Console is a powerful web-based development environment that enables you to write and execute SQL, PL/SQL, and JavaScript code. We'll show you how to sign up for Code Console and integrate it with your APEX workspace for enhanced development capabilities.
+* Creating an ASCVD Risk Calculator: In this main section of the workshop, we'll guide you step-by-step through building the ASCVD Risk Calculator application. You'll learn how to design the user interface, implement the necessary logic and calculations, and customize the application to meet your specific requirements.
 
 ### Prerequisites (Optional)
 
-*List the prerequisites for this lab using the format below. Fill in whatever knowledge, accounts, etc. is needed to complete the lab. **Do NOT list** each previous lab as a prerequisite.*
-
 This lab assumes you have:
 * An Oracle account
-* All previous labs successfully completed
-
-*This is the "fold" - below items are collapsed by default*
-
-In general, the Introduction does not have Steps.
-
-## Learn More
 
-*(optional - include links to docs, white papers, blogs, etc)*
+### Labs
+| # | Module                            | Estimated Time |
+|---|-----------------------------------|----------------|
+| 1 | Signing up for an APEX Workspace  | 10 minutes     |
+| 2 | APEX Application Setup            | 5 minutes      |
+| 3 | Signing up for Code Console       | 15 minutes     |
+| 4 | Creating an ASCVD Risk Calculator | 60 minutes     |
 
-* [URL text 1](http://docs.oracle.com)
-* [URL text 2](http://docs.oracle.com)
+### Let's get started!
 
 ## Acknowledgements
-* **Author** - <Name, Title, Group>
-* **Contributors** -  <Name, Group> -- optional
-* **Last Updated By/Date** - <Name, Month Year>
+* **Author** - Basarab Alexandru, Iordache Mircea
```

Comment:
no need to give me credit for it, it's all you :)

Reply:
I'll leave your name there since you also contributed to the entire workshop and the introduction wlks the users through all of it